# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1276352435 25200
#      Sat Jun 12 07:20:35 2010 -0700
# Node ID da51afafca807a5696e94fdfd45a3d1da50dedab
# Parent  fa00ccf7b1f9a6a3cdcd415b97f87a4808aabce6
scatter.m: Add new calling forms of function to documentation.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2010-06-12  Rik <octave@nomad.inbox5.com>
+
+        * plot/scatter.m: Add new calling forms of function to documentation.
+
 2010-06-10  Ben Abbott <bpabbott@mac.com>
 
 	* plot/private/__scatter__.m, plot/scatter.m: Honor Matlab color
 	settings.
 	* general/prepad.m: Fix typo 'dimm' -> 'dim'.
 
 2010-06-10  Jaroslav Hajek  <highegg@gmail.com>
 
diff --git a/scripts/plot/scatter.m b/scripts/plot/scatter.m
--- a/scripts/plot/scatter.m
+++ b/scripts/plot/scatter.m
@@ -12,20 +12,23 @@
 ## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 ## General Public License for more details.
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
-## @deftypefn {Function File} {} scatter (@var{x}, @var{y}, @var{s}, @var{c})
-## @deftypefnx {Function File} {} scatter (@dots{}, 'filled')
-## @deftypefnx {Function File} {} scatter (@dots{}, @var{style})
-## @deftypefnx {Function File} {} scatter (@dots{}, @var{prop}, @var{val})
+## @deftypefn {Function File} {} scatter (@var{x}, @var{y})
+## @deftypefnx {Function File} {} scatter (@var{x}, @var{y}, @var{s})
+## @deftypefnx {Function File} {} scatter (@var{x}, @var{y}, @var{c})
+## @deftypefnx {Function File} {} scatter (@var{x}, @var{y}, @var{s}, @var{c})
+## @deftypefnx {Function File} {} scatter (@var{x}, @var{y}, @var{s}, @var{c}, 'filled')
+## @deftypefnx {Function File} {} scatter (@var{x}, @var{y}, @var{s}, @var{c}, @var{style})
+## @deftypefnx {Function File} {} scatter (@var{x}, @var{y}, @var{s}, @var{c}, @var{prop}, @var{val})
 ## @deftypefnx {Function File} {} scatter (@var{h}, @dots{})
 ## @deftypefnx {Function File} {@var{h} =} scatter (@dots{})
 ##
 ## Plot a scatter plot of the data.  A marker is plotted at each point 
 ## defined by the points in the vectors @var{x} and @var{y}.  The size of
 ## the markers used is determined by the @var{s}, which can be a scalar, 
 ## a vector of the same length of @var{x} and @var{y}.  If @var{s} is not 
 ## given or is an empty matrix, then the default value of 8 points is used.
